pastas/check.py

```diff
@@ -0,0 +1,550 @@
+import logging
+from typing import Callable, Optional, Union
+
+import numpy as np
+from IPython.display import display
```

Comment:
i don't think IPython is a dependency

Reply:
Isn't automatically installed if you install jupyter?

Reply:
Yes but `jupyter` is not a dependency either.